server/src/routes/api/v1/squidsRouter.js

```diff
@@ -0,0 +1,14 @@
+import express from "express";
+
+import { Squid } from "../../../models/index.js";
+
+export const squidsRouter = new express.Router();
+
+squidsRouter.get("/", async (req, res) => {
```

Comment:
In the latest incarnation of the template, I've added a nextWrapper -- if you pull that in here, you can avoid the try/catch block